com.anydesk.Anydesk.appdata.xml

```diff
@@ -1,21 +1,24 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <component type="desktop">
   <id>com.anydesk.Anydesk.desktop</id>
-  <name>Anydesk</name>
+  <name>AnyDesk</name>
   <project_license>LicenseRef-proprietary</project_license>
-  <developer_name>Anydesk</developer_name>
-  <summary>Your desk at any desk</summary>
+  <developer_name>AnyDesk</developer_name>
+  <summary>Connect to a computer remotely</summary>
+  <summary xml:lang="pt_BR">Conectar com um computador remotamente</summary>
+  <summary xml:lang="es">Conéctese a un ordenador de forma remota</summary>
   <metadata_license>CC0-1.0</metadata_license>
   <url type="homepage">https://www.anydesk.com</url>
-  <description>
-    <p>Working together? Accessing your apps from the road? Showing family videos to your friends?
-    No problem with AnyDesk. And your data stays at home.</p>
-    <p>AnyDesk is the fastest remote desktop software on the market. It allows for new usage scenarios and applications that have not been possible with current remote desktop software.</p>
-  </description>
+  <description><p>AnyDesk ensures secure and reliable remote desktop connections for IT professionals and on-the-go individuals alike.</p></description>
```

Comment:
I think we should put the following language into a new paragraph here as well (similar to the other extra-data applications.)

`<p>NOTE: This wrapper is not verified by, affiliated with, or supported by AnyDesk.</p>`